projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -149,8 +164,23 @@ export const mapDispatchToProps = ( dispatch, featureSlug ) => {
 	}
 };
 
-export const getStepContent = stepSlug => {
+export const getStepContent = ( state, stepSlug ) => {
```

Comment:
Was just wondering what reason do we have for sending state to this function? I don't see it being used anywhere

Reply:
Ah, good catch! This was left-over from an implementation that I changed directions on and missed this in the cleanup.